c10/util/Logging.cpp

```diff
@@ -384,7 +384,7 @@ void initLogging() {
   detail::setLogLevelFlagFromEnv();
 }
 
-bool InitCaffeLogging(int* argc, char** argv) {
+bool InitCaffeLogging(const int* argc, char** argv) {
```

Comment:
These can't be const right? They are initializing global loggers

Reply:
argc is argument number, making it const is intuitive.